### ISSUE TITLE
Support use when no TTY is present

### ISSUE THREAD

**Issue by wagoodman** [wagoodman/bashful#58]

**Comment by wagoodman** [comment]
@karuppiah7890  Yeah, I meant to write up a description... but essentially, yes, it would be to disable the fancy terminal tricks and just output what is being run. When you run this from the repo root, for instance:
```
docker run  -v $(pwd)/example:/example -v $(pwd)/scripts:/scripts  -w / karuppiah7890/bashful:latest bashful run example/00-demo.yml
```
A few artifacts appear. It would be ideal to have the option to eliminate any ansi control character usage entirely when used in a CI setting.

**Comment by karuppiah7890** [comment]
Yeah, you are right. When I tried running it in Gitlab CI with a test repo,
the output wasn't very neat as it couldn't get the fancy stuff
